website/content/docs/job-specification/disk_throttle.mdx

```diff
@@ -0,0 +1,113 @@
+---
+layout: docs
+page_title: disk_throttles block in the job specification
+description: |-
+  Configure disk I/O throttling limits in the `disk_throttles` block of the Nomad job specification.
+---
+
+
+# `disk_throttle` block in the job specification
+
+
+<Placement groups={['job', 'group', 'task', 'resources', 'disk_throttle']} />
+
+The disk_throttle block is used to set limits on the disk I/O (Input/Output) a task can perform on a specific block device. 
+This block helps mitigate the "noisy neighbor" problem, where a single task consuming excessive disk bandwidth can negatively impact other tasks running on the same node.
+
+When a disk_throttle block is added, Nomad will limit the task's I/O throughput in bytes per second (BPS) or I/O operations per second (IOPS). 
+This feature is primarily implemented using Linux cgroups.
+
+
+```hcl
+job "docs" {
+  group "example" {
+    task "database" {
+      resources {
+        disk_throttle {
+          # Major/minor number of the device (e.g., /dev/sda is 8:0)
+          major      = 8
+          minor      = 0
+
+          # Limit read/write speed to 10 MB/s
+          read_bps   = 10485760
+          write_bps  = 10485760
+
+          # Limit read/write operations to 100 per second
+          read_iops  = 100
+          write_iops = 100
+        }
+      }
+    }
+  }
+}
+```
+
+In the example above, the database task is subject to I/O limits on the device with major number 8 and minor number 0. 
```

Comment:
```suggestion
In the preceding example, the database task is subject to I/O limits on the device with major number 8 and minor number 0. 
```

We avoid using directional words like above and below. Instead:
- above -> preceding
- below -> following